### ISSUE TITLE
Contact details not scrollable

### ISSUE THREAD

**Issue by tbleher** [nextcloud/contacts#2939]
### Describe the bug

Since updating nextcloud from version 23 to version 24 (v24.0.5, to be precise), I can no longer scroll the contact details. Contrary to previous versions, contact details are shown as a single column, which is not scrollable. This means that only parts of the contact details can be seen and edited.

This happens both on Firefox (104.0.2) and Google Chrome (102.0.5005.61) on Ubuntu Jammy.

Looking around using Google Chrome's DevTools, this seems to be caused by the following CSS rule:

```
.content {
	&:not(.with-sidebar--full) {
		position: fixed;
	}
}
```
which is found in `NcContent.vue`. Disabling this rule makes the view work as desired.

### Steps to reproduce

1. Go to 'Contacts App'
2. Select a Contact which doesn't fit on the screen
3. Try to scroll down to see all contact details

### Expected behavior

Scrolling should be enabled, so all details can be viewed.

### Actual behavior

Scrolling is prohibited by CSS rules, and thus not all details are shown, nor can they be edited.

### Contact version

5.0.0-alpha.1

(Note: I did not update to 5.0.0-alpha.1 on purpose - NextCloud proposed to update the app to this version; is this alpha version supported?)

### Operating system

Ubuntu 20.04

### PHP engine version

PHP 7.4

### Web server

Apache (supported)

### Database

MariaDB

**Comment by GretaD** [comment]
Hi @tbleher,

We are really sorry about this, there was a problem with v5.0.0-alpha1 release, we fixed it but you need to downgrade to v4.2.1 to have contacts in a working state again.

To do that, you need to disable the contacts app, then delete contacts from the apps/ folder. Download the old version from: https://apps.nextcloud.com/apps/contacts/releases?platform=24#24
and then re-enable the app. This should work, but if you face any problem, let us know.
Sorry again,
Greta

**Comment by tbleher** [comment]
@GretaD Thank you very much for the quick response! Your suggestion worked :+1:  :)
